_tools/check_docs.ts

```diff
@@ -23,131 +23,38 @@ import {
   type Location,
   type TsTypeDef,
 } from "@deno/doc";
-import { pooledMap } from "@std/async/pool";
+import { walk } from "../fs/walk.ts";
+import { isTestFile } from "./utils.ts";
+import { greaterOrEqual, parse } from "@std/semver";
+import { join } from "../path/join.ts";
```

Comment:
Not that it really matters. More for consistency.
```suggestion
import { walk } from "@std/fs/walk";
import { isTestFile } from "./utils.ts";
import { greaterOrEqual, parse } from "@std/semver";
import { join } from "@std/path/join";
```